applications/sdp/gpio/README.rst

```diff
@@ -0,0 +1,25 @@
+.. _sdp_gpio:
+
+SDP GPIO
+########
+
+This application demonstrates how to use SDP on FLPR in Zephyr threadless mode, by implementing a subset of Zephyr GPIO API.
```

Comment:
normally we should provide a definition for the abbreviation (software defined peripherals), but last time the decision was to avoid doing so - is this still the case?

Reply:
Yes